### ISSUE TITLE
Editing installerscript.qs in template doesnt effect deployment

### ISSUE THREAD

**Issue by sufuk** [QuasarApp/CQtDeployer#567]
Description:
I changed to installerscript to disabling shortcuts for my one of bin files but cqtdeployer seems does not seem using  it all.

**To Reproduce**
1. Create a deploy.json file like this;
```
    "bin": ["./bin/executable1", "./bin/executable2"],
    "noOverwrite": false,
    "qmake": "/usr/lib/qt5/bin/qmake",
    "force-clear": true,
    "runScript": ["executable1;executable1.sh", "executable2;executable2.sh"],
    "targetPackage": "MyApp"
```
2. Run this command in terminal;

```
cqtdeployer -confFile deploy.json qif getDefaultTemplate -targetDir template
```
3. Change the installscript.qs in template/defaultQIFWTemplate you can change or even delete its content it doesn't effect anything (it should) 

4. To test run following command:

```
cqtdeployer -confFile deploy.json qif template/defaultQIFWTemplate/ -targetDir Distribution
```
5. Install the file you' ll see any change in installscript.qs did not effect anything.

**Expected behavior**
Well when changed installscript.qs it should effect the whole thing :)
 

**Desktop:**
 - OS: Ubuntu
 - Version  20.04

**Extras:**

-Even Added console.log,("something") but doesnt show up anywhere,
-Deleted content of installscript.qs but doesnt effect anything.
- I dont think cqtdeployer does not use installscript.qs when we specify a template.
-  added .json file as a txt in attachment because its not supported in githb.

[deploy.txt](https://github.com/QuasarApp/CQtDeployer/files/6301663/deploy.txt)
[deploylog.txt](https://github.com/QuasarApp/CQtDeployer/files/6301665/deploylog.txt)
[templatelog.txt](https://github.com/QuasarApp/CQtDeployer/files/6301666/templatelog.txt)

**Comment by EndrII** [comment]
Thanks for your PR. 
i think this issue will be fixed today.

**Comment by sufuk** [comment]
Thank youuuu.

Andrei Yankovich ***@***.***>, 13 Nis 2021 Sal, 09:33
tarihinde şunu yazdı:

**Comment by EndrII** [comment]
@sufuk 
Bad news, i con not reproduse this bug.
i created new test case for this issue, but all is works correctly. If you want, you can check my new test case [this](https://github.com/QuasarApp/CQtDeployer/pull/565/commits/844fcbfbc59821d4cd3d07bfceadd0ab948df7b2#diff-74cff32ba690850207d64acad707232ae69b3fa7188487f0f78301711a910eeeR1213-R1270) 

Test case added in #565 PR.

**Comment by EndrII** [comment]
This bug is fixed in cqtdeployer 1.5.0.28 Please update.
